mlir/test/Dialect/Bufferization/Transforms/one-shot-bufferize.mlir

```diff
@@ -269,3 +269,14 @@ func.func @materialize_in_dest_raw(%f: f32, %f2: f32, %idx: index) -> (tensor<5x
 
   return %0, %r : tensor<5xf32>, f32
 }
+
+// -----
+
+// CHECK-LABEL: func @func_with_assert(
```

Comment:
This file does not run the buffer deallocation pass. This test should be in `Bufferization/Transforms/OwnershipBasedBufferDeallocation/misc-other.mlir`.